.orchestra/ci/install-python-dependencies.sh

```diff
@@ -1,6 +1,12 @@
 #!/bin/bash
+
 set -euo pipefail
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
-pip3 -q install --user --upgrade -r "${DIR}/requirements.txt"
+user_flag="--user"
+if [[ ! -z "${VIRTUAL_ENV:-}" ]]; then
+    user_flag=""
+fi
+
+pip3 -q install $user_flag --upgrade -r "${DIR}/requirements.txt"
```

Comment:
```suggestion
pip3 -q install "$USER_FLAG" --upgrade -r "${DIR}/requirements.txt"
```

Reply:
The lack of quotes is intended. `pip install ""` is not valid.

Reply:
Additionally, I usually only use uppercase for bash variables when they're constants / env.

Reply:
1. I wonder if you can use the array syntax with a regular variable and avoid this annoyance. Otherwise you can rename it to `FLAGS` and maybe in the future we'll add more stuff.
2. We always use them upper case.

Reply:
1. I actually can just use an array, IDK why I didn't think about it. Changed it to an array
2. Changed